charts/nydus-snapshotter/templates/nydus-snapshotter/snapshotter-daemonset.yaml

```diff
@@ -117,6 +117,13 @@ spec:
           toml set --overwrite $etcContainerd plugins.\"io.containerd.grpc.v1.cri\".containerd.disable_snapshot_annotations false
           toml set --overwrite $etcContainerd plugins.\"io.containerd.grpc.v1.cri\".containerd.snapshotter nydus
 
+          if grep -q "\[proxy_plugins\]" $etcContainerd; then
+            echo '[proxy_plugins.nydus]' >> $etcContainerd
+            echo '  type = "snapshot"' >>  $etcContainerd
+            echo '  address = "/run/containerd-nydus/containerd-nydus-grpc.sock"' >>  $etcContainerd
+            exit 0
+          fi
+
           # toml command not support to set block, so just use cat command.
```

Comment:
Thanks for your fix! But I feel like your code is duplicated with a few lines below?